content/docs/user-guide/external-dependencies.md

```diff
@@ -146,27 +146,39 @@ $ dvc run -n download_file \
 
 </details>
 
-## Example: DVC remote aliases
+## Example: Using DVC remote aliases
 
-If instead of a URL you'd like to use an alias that can be managed
-independently, or if the external dependency location requires access
-credentials, you may use `dvc remote add` to define this location as a DVC
-Remote, and then use a special URL with format `remote://{remote_name}/{path}`
-to define an external dependency.
+You may want to encapsulate external locations as configurable entities that can
+be managed independently. This is useful if multiple dependencies (or stages)
+reuse the same location, or if its likely to change in the future. And if the
+location requires authentication, you need a way to configure it in order to
+connect.
 
-For example, for an HTTPs remote/dependency:
+[DVC remotes](/doc/command-reference/remote) can do just this. You may use
+`dvc remote add` to define them, and then use a special URL with format
+`remote://{remote_name}/{path}` (remote alias) to define the external
+dependency.
+
+Let's see an example using SSH. First, register and configure the remote:
+
+```dvc
+$ dvc remote add myssh ssh://myserver.com
+$ dvc remote modify --local myssh user myuser
+$ dvc remote modify --local myssh password mypassword
+```
```

Comment:
This makes me realize our SSH examples for `remote add/modify` may be misleading (in other docs) because we sometimes use `ssh://user@example.com...` when adding, and then mention about modifying the credentials or even show an example of `remote modify example user ...` but I don't know if that will work correctly with DVC. Need to check...

Reply:
I tried testing this but can't because of https://github.com/iterative/dvc/issues/4712.